sources/api/pluto/src/eks.rs

```diff
@@ -46,8 +46,14 @@ where
     let config = sdk_config(region).await;
 
     let client = if let Some(https_proxy) = https_proxy {
-        let http_connector = proxy::setup_http_client(https_proxy, no_proxy)?;
-        let http_client = HyperClientBuilder::new().build(http_connector);
+        #[cfg(feature = "fips")]
+        let crypto_mode = CryptoMode::AwsLcFips;
+        #[cfg(not(feature = "fips"))]
+        let crypto_mode = CryptoMode::AwsLc;
+
+        let http_client = HyperClientBuilder::new()
+            .crypto_mode(crypto_mode)
+            .build_https();
```

Comment:
Don't we need this for the EC2 client also?

Reply:
I missed that, I'll update in a subsequent revision.